content/wiki/formatting_scheme.md

```diff
@@ -24,19 +24,32 @@ For those files, the formatting produced by `rustfmt` is required.
 The following rustfmt config is recommended:
 
 ```toml
-# LINEBENDER RUSTFMT CONFIG - v1
+# LINEBENDER RUSTFMT CONFIG - v2
 # Ensure lines end with \n even if the git configuration core.autocrlf is not set to true
 newline_style = "Unix"
 
 # `Foobar { foo, bar }` is more readable than `Foo { foo: foo, bar: bar }`
 use_field_init_shorthand = true
 
+# Forces let else blocks to always be their own line(s).
+# Enforcing this helps readers scan all early returns of a function at a glance.
+single_line_let_else_max_width = 0
```

Comment:
Copying my [previous review comment](https://github.com/linebender/linebender.github.io/pull/86#discussion_r1918696031) from the last time you submitted this:
> If we do decide to set this, I'd expect it to have the same value as `single_line_if_else_max_width`, I think.

Reply:
I disagree, for the reason exposed in the comment above.

I think

```rust
let foo = if x { y } else { z };
```

is fundamentally different from

```rust
let Some(foo) = x { y } else { return z };
```

and that, unlike the `if else` case, the `else return` case deserves to be spread out no matter its length.

**In general, I think a control-flow-altering statement should always be on its own line.**

(In theory you can do a single line `if x { y } else { return z }`, but only by treating the return statement as an expression, which is rare in practice.)

Reply:
Thank you for providing reasoning. I'm not completely sure I agree; for one thing, no style guide would ever have the formatting:
```rust
some_fallible_operation()
    ?
    .action()
    ?;
```

To be clear, my preference would just be to stick with the default, rather than matching the two at zero; I don't think it's a significant enough win, and IDEs can already highlight exit points for you in a much more complete manner

That being said, I'm not going to argue for it further; e.g. the Xilem repository which already has this set has literally no occurrences which would be formatted on a single line anyway.